vllm/v1/attention/backends/flash_attn.py

```diff
@@ -438,7 +438,8 @@ def forward(
             query: shape = [num_tokens, num_heads, head_size]
             key: shape = [num_tokens, num_kv_heads, head_size]
             value: shape = [num_tokens, num_kv_heads, head_size]
-            kv_cache = [2, num_blocks, block_size, num_kv_heads, head_size]
+            kv_cache: KV cache tensor with shape 
+                [2, num_blocks, block_size, num_kv_heads, head_size]
```

Comment:
@hmellor 
Which one is better?

```suggestion
            kv_cache: shape = [2, num_blocks, block_size, num_kv_heads, head_size]
```

Reply:
Probably `kv_cache: shape = [2, num_blocks, block_size, num_kv_heads, head_size]` to match the others. We already know it's KV cache because the variable is named well and we already know it's a tensor because of tye type hint.